frontend/src/components/account-settings.ts

```diff
@@ -310,7 +310,7 @@ export class AccountSettings extends LiteElement {
             id="newPassword"
             name="newPassword"
             type="password"
-            label="${msg("New password")}"
+            label="${msg("New password (8-64 characters)")}"
```

Comment:
How about using [help-text](https://shoelace.style/components/input#help-text) and `minlength` for this?

```
label="${msg("New password")}"
help-text=${msg("8-64 characters")}
minlength="8"
```

Reply:
Good call. Change applied across the different new password inputs